src/aap_eda/tasks/ruleset.py

```diff
@@ -208,6 +225,17 @@ def _perform_deactivate(
 
 # Started by the scheduler, executed by the default worker
 def monitor_activations() -> None:
+    job_id = "monitor_activations"
+    queue = get_queue("default")
+    if _job_in_queue(queue, job_id):
+        logger.info(
+            "Skip monitor_activations because another similar task is running"
```

Comment:
I think this msg is not accurate. Is not a "similar" but the exact same one. Not necessarily running, you are checking enqueued states. The id might be useful for debugging. 

```suggestion
            f"Skip monitor_activations task id {job_id} "
            "because another instance of this task is already enqueued."
```